### ISSUE TITLE
Leaving an unedited matrix field containing a super table field triggers "Leave site?" alert

### ISSUE THREAD

**Issue by jsunsawyer** [engram-design/SuperTable#424]
**Description**
I was checking some settings on various fields, and kept getting the `Leave Site? Changes you made may not be saved.` alert on a matrix field that contained a super table field.

Removing the super table field resolved this, and no other fields had issues, so it seems that something in this plugin is causing the "Leave site?" alert to trigger

**Steps to reproduce**
1. Create a matrix field containing a super table field
2. View field settings
3. Leave page

**Additional info**
- Plugin version: 2.6.8
- Craft version: 3.7.4
- Multi-site: No

**Comment by engram-design** [comment]
Just to clarify, this is when editing the field settings itself, right? If so, that seems unusual behaviour and not something I can replicate with the good few field combination I have with testing.

So you edit the Matrix field, touch nothing, and refresh or navigate away from the page and that warning shows?

**Comment by jsunsawyer** [comment]
@engram-design Exactly. I get the warning just from having the field load, touching absolutely nothing, then leaving the page.

**Comment by jsunsawyer** [comment]
Here's a video showing this happening. As you can see, it doesn't always happen, but does most of the time.

https://user-images.githubusercontent.com/1007191/127746817-2d4162e9-cfbc-4873-9494-a5dc4787d16a.mov

If I delete the Super Table field inside this matrix, this warning never shows up again.

**Comment by jsunsawyer** [comment]
My Super Table field in question is pretty simple.

- Field Layout: Matrix
- 3 Plain Text fields
- 1 Table Field (2 number inputs for coordinates)

I with only a single plain text field in the Super Table, and still had the issue. Removing from the Matrix field completely was the only thing that resolved things.

**Comment by engram-design** [comment]
Wondering if you might be able to export the field settings with [Field Manager](https://github.com/verbb/field-manager) so I can replicate?

**Comment by jsunsawyer** [comment]
[export.json.zip](https://github.com/verbb/super-table/files/6918168/export.json.zip)

Here ya go!

**Comment by engram-design** [comment]
I've just spun up this example, and didn't have the [Little Layout](https://github.com/wbrowar/craft-little-layout) plugin installed, so those fields didn't actually import. I thought to test it just without those fields, and didn't get the popup.

I then installed the Little Layout plugin, re-imported your field, and there we go - the "Leave site" popup appeared. To confirm, I removed all Little Layout fields, and this went away.

Seems like an issue with the [Little Layout](https://github.com/wbrowar/craft-little-layout) plugin.
